packages/vscode-extension/src/handlers.ts

```diff
@@ -360,12 +360,16 @@ export function getSystemInputs(): Inputs {
   return answers;
 }
 
-export async function createNewProjectHandler(args?: any[]): Promise<Result<null, FxError>> {
+export async function createNewProjectHandler(args?: any[]): Promise<Result<any, FxError>> {
   ExtTelemetry.sendTelemetryEvent(TelemetryEvent.CreateProjectStart, getTriggerFromProperty(args));
-  return await runCommand(Stage.create);
+  const result = await runCommand(Stage.create);
+  if (result.isOk()) {
+    commands.executeCommand("vscode.openFolder", result.value);
```

Comment:
So this will make sure all the telemetries been sending out before we open folder, right?

Reply:
Yes